### PR TITLE
MOB-3845 Add "Legal Notice" placeholder under T&C on the Checkout page

### PR DESCRIPTION
**Pull request by mateid-kr**
## JIRA
[MOB-3845](https://karhoo.atlassian.net/browse/MOB-3845)

## DESCRIPTION
 Add "Legal Notice" placeholder under T&C on the Checkout page

## Types of changes

What types of changes does your code introduce to the project?
_Put an `x` in the boxes that apply_

- [ ] Bugfix 
- [X] New feature 
- [ ] Tech Debt
- [ ] Breaking change 
- [ ] Documentation Update
- [ ] Other: *define what type of change it is*

## CHECKLIST

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. This is simply a reminder of what we are going to look for before merging code._
Please review the guidelines for contributing to this repository.

- [X] All work relating to the ticket is complete (Code complete/Acceptance Criteria met)
- [ ] Lint and unit tests pass locally (if appropriate)
- [ ] Added tests that prove the fix is effective or that feature works (if appropriate)
- [ ] Documentation (if appropriate)

## SCREENSHOTS
[Add screenshots if any for changes you have made to the App/UI]

### CODE REVIEW

File: uisdk/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -397,4 +397,11 @@
     <!--  Loyalty  -->
     <string name="kh_uisdk_loyalty_balance_title">BALANCE: %s POINTS</string>
     <string name="karhoo_uisdk_locale">en_GB</string>
+
+    <!-- Legal Notice  -->
+    <string name="kh_uisdk_legal_notice_label">Legal Notices</string>
+    <string name="kh_uisdk_legal_notice_title">Karhoo</string>
+    <string name="kh_uisdk_legal_notice_link">mailto:test@karhoo.com</string>
+    <string name="kh_uisdk_legal_notice_text">The data collected is electronically processed by Karhoo sending email to %s. Your data may need to be translated</string>
```
Comment (mohadian) [inline uisdk/src/main/res/values/strings.xml:405]: We need to remove the value for these keys

File: uisdk/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -397,4 +397,11 @@
     <!--  Loyalty  -->
     <string name="kh_uisdk_loyalty_balance_title">BALANCE: %s POINTS</string>
     <string name="karhoo_uisdk_locale">en_GB</string>
+
+    <!-- Legal Notice - The strings can be translated by DP -->
+    <string name="kh_uisdk_legal_notice_label">Legal Notices</string>
+    <string name="kh_uisdk_legal_notice_title" translatable="false" />
```
Comment (mohadian) [inline uisdk/src/main/res/values/strings.xml:403]: The key should be `translatable`, can we add it to all locales? the value would be empty though. 
Did you add the `translatable="false"` or it was added by phrasee.com?

**Review by mohadian — COMMENTED** [review]
The PR looks fine, I also did a smoke test and it works fine.

File: uisdk/src/main/res/values/strings.xml (hunk 2 above)
Comment (mateid-kr) [reply uisdk/src/main/res/values/strings.xml:403]: The translatable="false" is only for the linter to not show a warning
Phrase removes empty strings from the locale files, so in order to not need to add them manually at each strings updates, i've added the translatable flag

**Review by mionescu95 — DISMISSED** [review]

**Review by mionescu95 — APPROVED** [review]

File: uisdk/src/main/res/values/strings.xml (hunk 2 above)
Comment (mateid-kr) [reply uisdk/src/main/res/values/strings.xml]: Found a way to export also the empty keys from Phrase, so i've removed the translatable flag

**Pull request merged**